cg/meta/workflow/mutant/metadata_parser/metadata_parser.py

```diff
@@ -0,0 +1,74 @@
+from cg.apps.lims.api import LimsAPI
+from cg.exc import CgError
+from cg.meta.workflow.mutant.metadata_parser.models import SampleMetadata, SamplesMetadataMetrics
+from cg.meta.workflow.mutant.metadata_parser.utils import (
+    get_internal_negative_control_id,
+    is_sample_external_negative_control,
+)
+from cg.models.cg_config import LOG, CGConfig
+from cg.store.store import Store
+from cg.store.models import Case, Sample
+
+
+class MetadataParser:
+    def __init__(self, config: CGConfig) -> None:
```

Comment:
Explicitly pass `Store` and `LimsAPI` as parameters in the constructor here instead of `CGConfig`. It is better to be explicit about what the class is composed of and needs in order to be instantiated.